source/plugins/go/src/oms.go

```diff
@@ -1124,6 +1126,20 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 	}
 	DataUpdateMutex.Unlock()
 
+	if IsAADMSIAuthMode == true && !IsGenevaLogsIntegrationEnabled {
```

Comment:
1. you would need to add check for !IsWindows as well since this specific to linux. updated condition will be  if IsAADMSIAuthMode == true && !IsGenevaLogsIntegrationEnabled && !IsWindows 
2. Move this logic before this line if ContainerLogSchemaV2 == true || ContainerLogsRouteADX == true {

Reply:
changed